src/data.ts

```diff
@@ -134,25 +134,30 @@ async function getLinkedNotes(source_id:string, maxDegree:number) : Promise<Map<
     visited.push(...pending);
     pending = [];
 
-    notes.forEach(note => {
+    for (const note of notes) {
       // store note data to be returned at the end of the traversal
-      const links = getAllLinksForNote(note.body);
+      const forwardLinks = getAllLinksForNote(note.body);
       noteMap.set(note.id, {
         id: note.id,
         title: note.title,
         parent_id: note.parent_id,
-        links: links});
+        links: forwardLinks,
+      });
 
-      // stash any new links for the next iteration
-      links.forEach(link => {
+      const allLinks = [
+        ...forwardLinks,
+        ...(await getAllBacklinksForNote(note.id)),
```

Comment:
Please put this backlinks feature behind a setting, disabled by default. Same with the `marker` arrow.

Reply:
Good idea.